packages/core/strapi/lib/Strapi.js

```diff
@@ -460,7 +450,9 @@ class Strapi {
     }
 
     // admin
-    await this.admin[lifecycleName]({ strapi: this });
+    if (_.has(this, 'admin')) {
```

Comment:
this check is not necessary anymore. you can check that this.admin has the lifecycle however